taxi/models.py

```diff
@@ -20,6 +20,9 @@ class Driver(AbstractUser):
     class Meta:
         ordering = ("username",)
 
+    def __str__(self) -> str:
+        return f"{self.username} ({self.first_name} {self.last_name})"
```

Comment:
It was optional here to add str method, because u inherit from AbstractUser. Sorry, I missed that in my first review 🤗 But it's also good!!